src/generators/schema/webpage.php

```diff
@@ -45,7 +46,7 @@ public function generate() {
 			}
 		}
 
-		$this->add_image( $data );
+		$this->add_images( $data );
```

Comment:
Same as above, on line 56 $data = $this-> is also used. I think we should do the same